file/writable_file_writer.h

```diff
@@ -256,6 +256,8 @@ class WritableFileWriter {
   // returns NotSupported status.
   IOStatus SyncWithoutFlush(const IOOptions& opts, bool use_fsync);
 
+  // Size including unflushed data written to this writer. If the next op is
```

Comment:
Is the "unflushed" here the same as "unsynced" in other places?

Reply:
No. WritableFileWriter has an internal buffer and unflushed data can include more than what has been written to the FileSystem.